app/views/application/_menu.html.haml

```diff
@@ -1,3 +1,4 @@
 %ul.menu
   %li{ class: active_if_on('books') }= link_to 'Books', books_path
-  %li{ class: active_if_on('authors') }= link_to 'Authors_path', authors_path
\ No newline at end of file
+  %li{ class: active_if_on('authors') }= link_to 'Authors', authors_path
+  %li{ class: active_if_on('lists') }= link_to 'Lists', lists_path
```

Comment:
@wildfiler Github sad there is no newline at the end of file. You should  add it